### ISSUE TITLE
Mouse cursor positioning is different for space character

### ISSUE THREAD

**Issue by inkychris** [microsoft/vscode#153706]
Issue Type: <b>Bug</b>

When using the mouse to position the cursor, usually the centre of the character is used to determine whether to place the cursor before or after the character. The space character is different, in that the cursor is always places to the left of the character. This makes placing the cursor at the begining of a space indented character difficult, often resulting in the cursor ending up infront of the final indenting space.

This is easy to see when enabling visual white spaces. You would expect to place the cursor between the two dots when clicking there but that only happens if you click at least half way between the dots.

VS Code version: Code 1.68.1 (30d9c6cd9483b2cc586687151bcbcd635f373630, 2022-06-14T12:48:58.283Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No

Extensions: none

**Comment by alexdima** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/152819 . @inkychris The fix for this is already out in [VS Code Insiders](https://code.visualstudio.com/insiders/) which can be installed in parallel to VS Code Stable. Would you mind trying it out and confirming that the fix works. Thank you!

**Comment by inkychris** [comment]
Oh sorry, I tried for a while to find an existing issue but... "caret" vs. "cursor" 🤦‍♂️. Insider does in fact fix the issue, looking forward to the next release!
